Description: Do not run the CUPS_EUC_JP test case on BSD/Hurd either, as glibc's iconv behavior is different than what the test expects.
Bug-Debian: http://bugs.debian.org/662996

--- a/cups/testi18n.c
+++ b/cups/testi18n.c
@@ -423,7 +423,7 @@
       puts("PASS");
   }
 
-#ifndef __linux
+#if !defined(__linux__) && !defined(__GLIBC__)
   fputs("cupsCharsetToUTF8(CUPS_EUC_JP): ", stdout);
 
   strcpy(legsrc, legdest);
